obs_youtube_uploader/web/index.html

```
@@ -196,6 +203,170 @@ <h2>Discord (combat logs)</h2>
+
+      <section class="card">
+        <h2>Root</h2>
+        <div class="row">
+          <span class="lab">Root mode</span>
+          <span class="hint" id="eve-root-mode">Not set</span>
+        </div>
+        <div class="row">
+          <span class="lab">Set root</span>
+          <input class="field mono" id="eve-root-input" type="text"
+                 spellcheck="false" placeholder="J123456">
+          <button class="btn" id="eve-set-root">Set</button>
```

HTMLHint (1.9.2)

[warning] The type attribute must be present on <button> elements.

(button-type-require)